parley/src/layout/cursor.rs

```diff
@@ -101,128 +101,135 @@ impl Cursor {
         self.affinity
     }
 
-    /// Returns a new cursor that is positioned at the next cluster boundary
+    /// Returns a new cursor that is positioned at the previous cluster boundary
```

Comment:
Is there a reason for swapping the order of these? it makes it kinda messy to read the diff.